src/projects/EnsureThat/StringArg.cs

```diff
@@ -42,8 +42,9 @@ public string IsNotEmpty(string value, string paramName = Param.DefaultName)
         {
             if (!Ensure.IsActive)
                 return value;
-
-            if (string.Empty.Equals(value))
+            
+            // !null && (null || empty) == !null && empty
+            if (value != null && string.IsNullOrEmpty(value))
```

Comment:
Remove comment.

Use the length instead:

```
if (value?.Length == 0)
```

Reply:
Exactly! I was tempted but not sure if it read well enough for a sign off :) Will update!